### ISSUE TITLE
subdomain request

### ISSUE THREAD

**Issue by tcgriffith** [rbind/support#54]
Hi, yihui,

I've recently played with blogdown and hugo a lot and I feel comfortable to apply for a subdomain in rbind now.

i request `tc.rbind.io`, my current netlify site is `tctc.netlify.com`

By the way, love the xmin theme! But i couldn't get disqus working right now(it's working in default theme). I'll experiment a bit more to find the problem.

**Comment by yihui** [comment]
Done!

You need to configure disqusShortname in config.toml: https://github.com/yihui/hugo-xmin/pull/4/files

**Comment by tcgriffith** [comment]
#40 
I wrote to netlify support and a guy helped me to set it up.
it is up and running now.

**Comment by yihui** [comment]
Your disqus config is incorrect here: https://github.com/rbind/blogsite/blob/master/layouts/partials/foot_custom.html

Please refer to https://github.com/yihui/hugo-xmin/blob/feature/disqus/exampleSite/layouts/partials/foot_custom.html

**Comment by tcgriffith** [comment]
@yihui many thanks for the help even when i didn't file an issue for that  （´͈ ᵕ `͈

**Comment by tcgriffith** [comment]
hi, @yihui  , I've tried to setup a new site from scratch, but still, disqus is not working.

What I've tried:

***

- start site  
    - started a new site using `blogdown::new_site()` in Rstudio  
    - installed `hugo-xmin` by `git submodule add https://github.com/yihui/hugo-xmin.git themes/hugo-xmin`  
- setup disqus
    - initial netlify deploy succeeded, then I setup a new disqus site by using the netlify domain. 
    - copied  `config.toml` from `themes/hugo-xmin/exampleSite/config.toml` to project dir, disqus short name changed accordingly in `config.toml`  
    - changed `layouts/partials/foot_custom.html` 
- deploy  
    - local:   it shows `Disqus comments not available by default when the website is previewed locally.`, so I assume it's normal.  
    - netlify: no problem in deploying. I used hugo version 0.25.1. However, the example site is displayed correctly,  except disqus  

***

Basically I've only touched config.toml and foot_custom.html
my local and netlify hugo version are both 0.25.1

For an reproducible example,  [here is the repo](https://github.com/tcgriffith/blogtest2) and [here is the netlify site](http://testtc.netlify.com/), could you take a look?

**Comment by yihui** [comment]
The Hugo config looks correct to me. There might be something wrong with the account `tc_testit` 
 on Disqus: https://docs.disqus.com/help/83/

**Comment by tcgriffith** [comment]
@yihui  I am happy that disqus is up and running now. But I'm not sure the reason. 
Maybe it's because I added `rbind.io` in the trusted domain in disqus configuration, and it may take some time for disqus to function properly.

Anyway if you don't mind this issue can be closed!

**Comment by yihui** [comment]
Great. Thanks for letting me know!
